checks.d/dns_check.py

```diff
@@ -7,6 +8,10 @@
 # project
 from checks import AgentCheck
 
+if sys.platform == 'win32':
+    from dns.rdtypes.ANY import *  # noqa
+    from dns.rdtypes.IN import *  # noqa
+
```

Comment:
If we don't do this an exception is thrown because on windows they dynamic type resolution fails. Don't ask -  https://github.com/rthalley/dnspython/issues/39